source/05-delivery/index.rst

```diff
@@ -7,32 +7,40 @@
 Delivery of COUNTER Reports
 ===========================
 
-Report providers MUST make tabular versions of COUNTER reports available from an administrative/reporting site accessible by members of the institution requesting the report. All COUNTER reports provided by the report provider MUST also be available via the COUNTER_SUSHI API. Delivery requirements are:
+Report providers MUST make COUNTER Reports and Standard Views of the COUNTER Reports available in JSON format via the COUNTER_SUSHI API. Report providers MUST also make tabular versions of their reports available from an administrative/reporting site accessible by members of the institution requesting the report.
 
-* Reports MUST be provided in the following formats:
+* Reports MUST be provided monthly.
+* Data MUST be updated within 4 weeks of the end of the reporting period.
+* Usage MUST be processed for the entire month before any usage for that month can be included in reports. If usage for a given month is not available yet, the report provider MUST NOT return usage for that month and MUST include exception 3031 in the report/response to indicate that usage is not ready for requested dates.
+* A minimum of the current year plus the prior 24 months of usage data MUST be available, unless the report provider is newly COUNTER compliant.
+* When report providers become compliant with a new release of the Code of Practice, they begin compiling usage compliant with the new release from the time they become compliant; and they MUST continue to provide the older usage that complies with the previous release(s) of the Code of Practice to fulfil the requirement.
+* The reports MUST allow the report consumer the flexibility to specify a date range, in terms of months, within the most recent 24-month period.
+* Report providers should provide reports on a per-customer basis. That is, if a business school has a separate customer ID from its parent university, the school and the university should be sent separate COUNTER reports.
```

Comment:
That is very problematic. In my experience the result will be that parts of the university usage also end up in the business school report, and some of the usage only ends up in the business school report. As a consequence it will be difficult or even impossible to determine the total usage for all subscriptions within the university. What actually would be required in such a situation would be a parent customer ID that allows to fetch a consolidated/deduplicated report, i.e. something like a consortium setup.